app/common/constants/ledgerStatuses.js

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const statuses = {
+  CORRUPTED: 'corruptedSeed',
```

Comment:
can this be more specific? ex: `CORRUPTED_SEED`? in case there may be other types of corruption (not that I'm advocating for more corruption!) 😛

Reply:
next one will be `CORRUPTED_DOOH` 😎 

yeah I can change it